public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -215,16 +207,30 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       };
     });
 
-    const dataFrames = increasingInterval({ startPeriod: 100, endPeriod: 1000, step: 300 }).pipe(
+    const responses = increasingInterval({ startPeriod: 100, endPeriod: 1000, step: 300 }).pipe(
       concatMap((_) => this.makeLogActionRequest('GetQueryResults', queryParams, queryFn)),
       repeat(),
       share()
     );
 
+    let errorsFromGetQuery: DataQueryError[] = [];
+    const dataFrames: Observable<DataFrame[]> = responses.pipe(
+      map((response) => {
+        // TODO: it's not entirely clear to me why but this map gets called twice, but the responses are the same
```

Comment:
I am not sure yet if I'm adding this bug or if I'm just moving it around. If anyone can see what I'm missing feel free to shout it out, it definitely feels wrong/hacky atm